test/integration/core/fs.tap.js

```diff
@@ -13,6 +13,8 @@ const helper = require('../../lib/agent_helper')
 const verifySegments = require('./verify')
 const NAMES = require('../../../lib/metrics/names')
 
+const nodeMajor = parseInt(process.version.split('.')[0].replace('v', ''), 10)
```

Comment:
this is fine but you could just use `semver` package.  

```js
const isGlobSupported = semver.satisfies(process.version, '>=22.0.0')
```

Reply:
We don't have that in our dev dependencies and it seems a bit excessive for this easy check.

Reply:
we totally have semver it's all over but i understand if you don't want to update

Reply:
Ah! It's in the regular dependencies list. I'll update.